pkg/api/helpers/upload.go

```diff
@@ -20,8 +20,21 @@ import (
 	"github.com/treeverse/lakefs/pkg/api/apigen"
 	"github.com/treeverse/lakefs/pkg/api/apiutil"
 	"github.com/treeverse/lakefs/pkg/httputil"
+	"golang.org/x/sync/errgroup"
 )
 
+// MaxUploadParts is the maximum allowed number of parts in a multi-part upload
+const MaxUploadParts int32 = 10000
+
+// MinUploadPartSize is the minimum allowed part size when uploading a part
+const MinUploadPartSize int64 = 1024 * 1024 * 5
+
+// DefaultUploadPartSize is the default part size to buffer chunks of a payload into
+const DefaultUploadPartSize = MinUploadPartSize
+
+// DefaultUploadConcurrency is the default number of goroutines to spin up when uploading a multipart upload
+const DefaultUploadConcurrency = 5
```

Comment:
Is it enough? Seems too low for me

Reply:
The is the default aws sdk uses - I'll make it configurable for lakectl fs upload

Reply:
If it's AWS defaults then it's good enough